src/headlessTask/index.js

```diff
@@ -99,13 +116,12 @@ const HeadlessTask = async (taskData) => {
       }
 
       if (taskData.eventText === '') {
-        const noticeIds = getNoticeIds(
-          matchingContexts,
-          eventMessageFromChromeURL
-        );
+        await getNoticeIds(eventMessageFromChromeURL);
```

Comment:
Same thing here … `getNoticeIds` could return the response and we can receive it here, instead of needing another global variable …